### PR TITLE
Functionality for changing password

### PR DESCRIPTION
**Pull request by YAWAL**
Functionality for changing password has been added into validateMessage function on the server

### CODE REVIEW

File: server/core/server-core.go [hunk 1]
```diff
@@ -110,8 +111,14 @@ func validateMessage(message *messageService.Message, messageType int, conn *web
...
-
-		//run change_pass function
+		ok, err := settingService.ChangePass(message)
+		if err != nil {
+			loger.Log.Errorf(" Error has occurred : ", err)
+		}
+		if !ok {
+			loger.Log.Warnf(" Password has not changed. Please try again")
+		}
+		return
```
Comment (bohdanlisovskyi) [inline server/core/server-core.go:121]: you need to return some status to client, is it change pass or not!
Example:
```
err :=  messageService.Message {
			Header:messageService.MessageHeader{
				Type_: "change_pass",
				Command:"Password has not changed. Please try again",
			},
			Body:message.Body,
		}

byteError, _ := son.Marshal(err)

conn.WriteMessage(messageType, byteError)
```

**Review by bohdanlisovskyi — APPROVED** [review]

**Review by 8tomat8 — APPROVED** [review]

**Comment by justakit** [comment]
So, when we try to change few fields (nickname, birthday etc.), we have to send separate message for each of them?

**Comment by YAWAL** [comment]
Dear Volodymyr,
Please, ignore this PR. According to changes in server code, this
functionality will be rewritten.

**Comment by YAWAL** [comment]
Architecture has been changed and this code does not confirm it anymore.

**Pull request closed without merging**